### PR TITLE
resolves #125. removing exception handling from the jax-rs services.

### LINKED ISSUE #125: Do not print a long stacktrace when developer throws a WebApplicationException
WebApplicationException is not an exceptional case in the execution of the program, but a way to control flow. Thus, I don't want to see a long stack trace when I throw `WebApplicationException(Response.Status.BAD_REQUEST)`, especially I might have already logged the problem in the data.

### PR DESCRIPTION
**Pull request by jadelkhoury**
resolves #125. removing exception handling from the jax-rs services.

See the branch for resulting code.
https://github.com/OSLC/lyo-adaptor-sample-modelling/tree/skipStacktrace

### CODE REVIEW

File: org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateResourceService.mtl [hunk 1]
```diff
@@ -298,35 +299,27 @@ import [javaClassFullName(aResource, anAdaptorInterface, null) /];
...
+            httpServletRequest.setAttribute("terms", terms);
+            final List<[dialogMethodResourceType(aSelectionDialog)/]> resources = [javaClassNameForAdaptorManager(anAdaptorInterface) /].[dialogMethodName(aSelectionDialog, true)/](httpServletRequest[commaSeparate(dialogMethodParameterList(aSelectionDialog, true), true, false)/], terms);
+            if (resources!= null) {
+                        httpServletRequest.setAttribute("resources", resources);
+                        RequestDispatcher rd = httpServletRequest.getRequestDispatcher("[resourceSelectorResultsJspRelativeFileName(aSelectionDialog) /]");
+                        rd.forward(httpServletRequest, httpServletResponse);
             }
-        } catch (Exception e) {
-            throw new WebApplicationException(e);
+            //a empty search should return an empty list and not NULL!
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateResourceService.mtl]: Isn't it a 500?

Comment (berezovskyi) [reply org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateResourceService.mtl]: should also be logged:

    log.error("A search should return an empty List and not 'null' if nothing was found!")

Comment (jadelkhoury) [reply org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateResourceService.mtl]: Please see new commit.

**Comment by berezovskyi** [comment]
Looks good

**Pull request merged**